app/src/api/auth/types.ts

```diff
@@ -0,0 +1,14 @@
+export type TUser = {
+  id?: string;
+  email?: string;
+  firstName?: string;
+  lastName?: string;
+  birthday?: string;
+  password?: string;
```

Comment:
It was optional because not all fields are necessary for signIn hook as parameters, due to i write another type with only necessary parameters(TUserParameters) and made in TUser type all fields not optional, right?